### PR TITLE
Add image dimensions node

### PR DESCRIPTION
**Pull request by joeyballentine**
image

### CODE REVIEW

File: backend/nodes/image_dim_nodes.py [hunk 1]
```diff
@@ -194,3 +194,36 @@ def run(
...
+        ]
+        self.category = IMAGE_DIMENSION
+        self.name = "Get Dimensions"
+        self.icon = "BsRulers"
+        self.sub = "Utility"
+
+    def run(
+        self,
+        img: np.ndarray,
+    ) -> any:
```
Comment (RunDevelopment) [inline backend/nodes/image_dim_nodes.py:225]: Why an `any` type hint? 

I don't know anything about py type hints, but wouldn't this stop the IDE from inferring the return type?

File: backend/nodes/image_dim_nodes.py [hunk 2]
```diff
@@ -194,3 +194,36 @@ def run(
...
+        super().__init__()
+        self.description = (
+            "Get the Height, Width, and number of Channels from an image."
+        )
+        self.inputs = [
+            ImageInput(),
+        ]
+        self.outputs = [
+            NumberOutput("Height"),
+            NumberOutput("Width"),
```
Comment (RunDevelopment) [inline backend/nodes/image_dim_nodes.py:214]: I think we should use the same order as the Resize node.

```suggestion
            NumberOutput("Width"),
            NumberOutput("Height"),
```

**Review by RunDevelopment — APPROVED** [review]

**Comment by RunDevelopment** [comment]
Oops, I meant to request changes, not approve.

File: backend/nodes/image_dim_nodes.py (hunk 1 above)
Comment (joeyballentine) [reply backend/nodes/image_dim_nodes.py:225]: Oh yeah, I forgot to finish that

**Pull request merged**